src/hooks/useAppBootstrap.js

```diff
@@ -1,72 +1,24 @@
-import Minilog from '@cozy/minilog'
 import { Linking, LogBox } from 'react-native'
 import { useEffect, useState } from 'react'
+import { deconstructCozyWebLinkWithSlug } from 'cozy-client'
 
 import { SentryCustomTags, setSentryTag } from '/libs/monitoring/Sentry'
 import { devConfig } from '/constants/dev-config'
 import { manageIconCache } from '/libs/functions/iconTable'
+import { getDefaultIconParams } from '/libs/functions/openApp'
 import { navigate } from '/libs/RootNavigation'
 import { routes } from '/constants/routes'
 import { useSplashScreen } from '/hooks/useSplashScreen'
 
-if (devConfig.ignoreLogBox) LogBox.ignoreAllLogs()
-
-const log = Minilog('useAppBootstrap')
-
-const parseOnboardingURL = url => {
-  try {
-    if (!url || !url.includes('onboarding')) {
-      return undefined
-    }
-
-    const onboardingUrl = new URL(url)
-    const onboardUrl = onboardingUrl.searchParams.get('onboard_url')
-    const fqdn = onboardingUrl.searchParams.get('fqdn')
-
-    if (!onboardUrl && !fqdn) {
-      return undefined
-    }
-
-    return {
-      onboardUrl,
-      fqdn
-    }
-  } catch (error) {
-    log.error(
-      `Something went wrong while trying to parse onboarding URL data: ${error.message}`
-    )
-    return undefined
-  }
-}
-
-const parseFallbackURL = url => {
-  const defaultParse = { fallback: undefined, root: routes.stack }
-
-  if (url === null) {
-    return defaultParse
-  }
-
-  try {
-    const makeURL = new URL(url)
-    const fallback = makeURL.searchParams.get('fallback')
-    const isHome = makeURL.pathname.split('/')[1] === 'home'
+import {
+  parseFallbackURL,
+  parseOnboardingURL
+} from './useAppBootstrap.functions'
```

Comment:
think we should still use aliased import

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/pull/550/commits/cd72b2044f0c573367989e1bbbd0dd2f1cad6805